app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_name/AddBusStopName.java

```diff
@@ -0,0 +1,50 @@
+package de.westnordost.streetcomplete.quests.bus_stop_name;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopName extends SimpleOverpassQuestType
+{
+
+	@Inject public AddBusStopName(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return " nodes, ways with !name and noname != yes" +
+			   " and (public_transport=platform or (highway=bus_stop and public_transport!=stop_position))";
```

Comment:
This would include any kind of public transport platform (i.e. also train stations and metro stops). 

I think it would make sense if the query is the same as for `AddBusStopShelter`:
https://github.com/westnordost/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_shelter/AddBusStopShelter.java#L24-L26

Reply:
As tram was named in the original issue, it was making sense to include them as well. So is it better to only show bus stops ?

Reply:
Well answer is below.